feature_engine/creation/math_features.py

```diff
@@ -130,6 +136,54 @@ class MathFeatures(BaseCreation):
     0   1   4         2.5
     1   2   5         3.5
     2   3   6         4.5
+
```

Comment:
All of this, should go in the user guide, not here. It is too much information for the class docstrings. But let's hold on with the change until we finalize the class update